spring-core/src/main/java/org/springframework/asm/SymbolTable.java

```diff
@@ -1473,7 +1473,7 @@ private static final class LabelEntry {
 
     /**
      * Another entry (and so on recursively) having the same hash code (modulo the size of {@link
-     * SymbolTable#labelEntries}}) as this one.
+     * SymbolTable#labelEntries}) as this one.
```

Comment:
Please note that this class belongs to our fork of the ASM project.

Consequently, we do not modify the Javadoc. Changes to ASM Javadoc should be made in the upstream project.

Thus, I will revert that when merging this PR.

Reply:
I created https://gitlab.ow2.org/asm/asm/-/merge_requests/424 to fix it.